curp/src/server/raw_curp/mod.rs

```diff
@@ -1369,6 +1411,7 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
                 (vec![], String::new(), false)
             }
         };
+        let _ig = self.ctx.cluster_info.cluster_version_inc();
```

Comment:
Should fall back the cluster version?

Suppose there are 10 conf change requests sent to a leader. Then the cluster version on this leader is updated to 10. A client fetches the cluster and updates its local cluster version to 10. Then the leader failed, and these conf change logs are reverted by a new elected leader. The above client with the cluster version of 10 will still keep the old cluster members and will not be updated quickly even the cluster members are actually changed.

Reply:
Fixed